src/main/java/org/isf/patconsensus/service/PatientConsensusIoOperation.java

```diff
@@ -0,0 +1,96 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.patconsensus.service;
+
+import java.util.Optional;
+
+import org.isf.opetype.model.OperationType;
+import org.isf.patconsensus.model.PatientConsensus;
+import org.isf.utils.db.TranslateOHServiceException;
+import org.isf.utils.exception.OHServiceException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(rollbackFor=OHServiceException.class)
+@TranslateOHServiceException
+public class PatientConsensusIoOperation {
+
+	@Autowired
+	private PatientConmsensusIoOperationRepository repository;
+
+	/**
+	 * Return the PatientConsensus if exists {@link PatientConsensus}s
+	 *
+	 * @return the {@link PatientConsensus}s. It could be <code>empty</code>.
+	 * @throws OHServiceException
+	 */
+	public Optional<PatientConsensus> getPatientConsensusByUserId(String userName) throws OHServiceException {
+		return repository.findByUserName(userName);
+	}
+
+	/**
+	 * Insert an {@link PatientConsensus} in the DB
+	 *
+	 * @param patientConsensus - the {@link PatientConsensus} to insert
+	 * @return <code>PatientConsensus</code> inserted.
+	 * @throws OHServiceException
+	 */
+	public PatientConsensus newPatientConsensus(PatientConsensus patientConsensus) throws OHServiceException {
+		return repository.save(patientConsensus);
+	}
+
+	/**
+	 * Update an {@link PatientConsensus}
+	 *
+	 * @param patientConsensus - the {@link PatientConsensus} to update
+	 * @return <code>PatientConsensus</code>.
+	 * @throws OHServiceException
+	 */
+	public PatientConsensus updatePatientConsensus(PatientConsensus operationType) throws OHServiceException {
+		return repository.save(operationType);
+	}
+
+	/**
+	 * Delete an {@link PatientConsensus}
+	 *
+	 * @param patientConsensus - the {@link PatientConsensus} to delete
+	 * @return <code>true</code> if the {@link PatientConsensus} has been delete, <code>false</code> otherwise.
+	 * @throws OHServiceException
+	 */
+	public boolean deletePatientConsensus(PatientConsensus patientConsensus) throws OHServiceException {
+		repository.delete(patientConsensus);
+		return true;
```

Comment:
I find this problematic.   I know the lots of the current code looks like this but we are slowly trying to clean it up.  The `delete` method has a `void` return.   Thus I feel this method should also return `void` and not force a `true` return value.